libraries/common/string.effekt

```diff
@@ -64,7 +64,17 @@ def repeat(str: String, n: Int): String = {
     if (n == 0) result
     else go(n - 1, result ++ str)
   }
-  go(n, "")
+  if(n > 0) { go(n, "") } else { "" }
```

Comment:
Couldn't we move the check into the recursive call?
Then it would be just `if (n <= 0) result` there and `go(n, "")` here.
I think it's isomorphic to your change, it just avoids the extra comparison

Reply:
Ah, yes, I was lazy when changing it 😅

Reply:
Works :)